services/prompter/prompt-utils.js

```diff
@@ -0,0 +1,13 @@
+class PromptUtils {
+  constructor(requests) {
+    this.requests = requests;
```

Comment:
This is what I thought, but, some sub prompters need it, and some other doesn't. Since not every child classes use this var, I don't want to set it upper in the class hierarchy. 

Moreover this abstract class has no use of these two variables. Since I'm not able to find a clear argument to make these variables upper, I prefer to keep them child side.